tests/security/test_scan_for_secrets.py

```
@@ -0,0 +1,527 @@
+
+        # Create safe file (example)
+        safe_file = tmp_path / ".env.example"
+        safe_file.write_text('API_KEY = "sk-' + "A" * 48 + '"')
+
+        import os
+
+        original_dir = os.getcwd()
+        os.chdir(tmp_path)
+
+        try:
+            critical, warnings = scan_repository()
```

Ruff (0.14.4)

Unpacked variable `critical` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)